imap_processing/tests/ultra/unit/test_ultra_l1a.py

```diff
@@ -219,7 +219,8 @@ def test_cdf_aux(
     dataset_aux = create_dataset({ULTRA_AUX.apid[0]: decom_ultra_aux})
     input_xarray_aux = load_cdf(test_data_path)
 
-    assert input_xarray_aux.attrs.keys() == dataset_aux.attrs.keys()
+    # write_cdf() injects some attributes that are not in the xarray
```

Comment:
We should really be testing explicit expectations I think, so it shouldn't matter if we are injecting extra attributes in write_cdf, we really want to know that the ones that were in the xarray dataset made it through to the output.